packages/vanilla/src/cells/TextAreaCell.tsx

```diff
@@ -44,6 +44,7 @@ export const TextAreaCell = (props: CellProps & VanillaRendererProps) => {
       id={id}
       disabled={!enabled}
       autoFocus={uischema.options && uischema.options.focus}
+      placeholder={uischema.options && uischema.options.placeholder}
```

Comment:
As you're changing the code here anyway: We should use the same `appliedUiSchemaOptions` approach like in `TextCell.tsx`.

Reply:
Changing it now...